### DISCUSSION TITLE
DNS settings t

### DISCUSSION THREAD

**Discussion by nathalienordstrom** [community/community#32011] [Other Feature Feedback, Questions, & Ideas]
I have a domain from another site and I want them to connect my domain to my GitHub webpage. They said
"In order for us to be able to point to your domain, we need to get the DNS settings to be added."

Where do I find those to send to them?

**Comment by airtower-luna** [comment]
Look at the [documentation for custom domains with Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site). :slightly_smiling_face:

**Reply by AdiXsus** [reply]
yep

**Comment by BLKKKBVSIK** [comment]
You can find your DNS Zones on the website where you bought your domain.
They are usually called "DNS Zones" or "DNS Servers", with two or more of them.
